railties/test/application/assets_test.rb

```diff
@@ -260,11 +260,11 @@ class User < ActiveRecord::Base; raise 'should not be reached'; end
       # Load app env
       app "production"
 
-      # Checking if Uglifier is defined we can know if Sprockets was reached or not
-      assert_not defined?(Uglifier)
+      # Checking if Terser is defined we can know if Sprockets was reached or not
+      assert_not defined?(Terser)
       get("/assets/#{asset_path}", {}, "HTTPS" => "on")
       assert_match "alert()", last_response.body
-      assert_not defined?(Uglifier)
+      assert_not defined?(Terser)
```

Comment:
I think these assertions aren't doing what we expect them to do any more. 

At one point, they were checking that Sprockets wasn't loading Uglifier to process `application.js`, but Uglifier is no longer in the rails/rails Gemfile (which these tests use) so I don't think these assertions will ever fail for Uglifier.

So changing from Uglifier to Terser would make sense because Terser _is_ in the Gemfile, however the app in the test isn't configured to use _any_ `js_compressor` at all now, so I don't think these assertions would fail now even if Sprockets was "compiling" the `application.js` file, which is what the test is supposed to be asserting.

Reply:
Then I think its worth of removing these assertions.

Reply:
Yes, I tried to change this with some random values, And it was passing without any failure. Then I think its worth of removing these assertions. WDYT?

Reply:
@skipkayhil I'm removing this change for now. Will create seperate PR for removing the test. WDYT?